src/Control/Carrier/Choose/Church.hs

```diff
@@ -50,7 +55,9 @@ instance Fail.MonadFail m => Fail.MonadFail (ChooseC m) where
 
 instance MonadFix m => MonadFix (ChooseC m) where
   mfix f = ChooseC $ \ fork leaf ->
-    mfix (runChoose (liftA2 Fork) (pure . Leaf)
+    mfix (runChoose
+      (liftA2 Fork)
+      (pure . Leaf)
       . f . fromJust . fold (Alt.<|>) Just)
```

Comment:
Can we use something more descriptive than `fromJust` here? `fromMaybe (“error: ChooseC mfix returned Just”)`?

Reply:
This is being replaced in #233 anyway, so I’mma leave it as-is.